cpp_test_suite/new_tests/cxx_group.cpp

```diff
@@ -827,6 +827,39 @@ class GroupTestSuite: public CxxTest::TestSuite
 		CxxTest::TangoPrinter::restore_unset("double_attr_value");
 	}
 
+	/* Verifies that a group can contain devices from a remote TANGO_HOST
+	 * (a Tango instance different from client's default TANGO_HOST).
+	 * An issue was reported when resolving names containing wildcards. */
+
+	void test_use_devices_from_remote_tango_host()
+	{
+		const std::string original_tango_host = std::getenv("TANGO_HOST");
+		const std::string external_tango_host = std::getenv("TANGO_HOST2");
```

Comment:
Thanks!

> This will throw an std::logic_error exception ("basic:string::_M_construct null not valid") if TANGO_HOST or TANGO_HOST2 environment variable is not defined.

I'd say this is exactly what we want. All I can do is to re-throw the exception with a better message. But this will come at the expense of test case readability (extra logic and conditions not important to the scenario). In my opinion latter is more important for developers.

Still it would be nice to get the backtrace, but this is something to be handled on the CxxTest side.

Reply:
> I'd say this is exactly what we want. All I can do is to re-throw the exception with a better message. But this will come at the expense of test case readability (extra logic and conditions not important to the scenario). In my opinion latter is more important for developers.

I agree with you and this is why I approved this PR. I think we don't need to do anything in this specific case.